test/spec/geo/ui/widgets/widgets_collection.spec.js

```diff
@@ -0,0 +1,30 @@
+var $ = require('jquery');
+var Model = require('cdb/core/model');
+var WidgetsCollection = require('cdb/geo/ui/widgets/widgets_collection');
+
+describe('widgets/widgets_collection', function() {
+
+  beforeEach(function() {
+    this.collection = new WidgetsCollection();
+  });
+
+  describe('bind', function() {
+
+    it('should disable categoryColors attribute if any other widget enables it', function() {
+      this.collection.reset([{ name: 'hello' }, { name: 'howdy' }, { name: 'hey', categoryColors: true }]);
+      var mdl1 = this.collection.at(0);
+      var mdl2 = this.collection.at(1);
+      var mdl3 = this.collection.at(2);
+      expect(mdl3.get('categoryColors')).toBeTruthy();
+      mdl1.set('categoryColors', true);
+      expect(mdl3.get('categoryColors')).toBeFalsy();
```

Comment:
I'd still add an assertion here to ensure that:
```
expect(mdl1.get('categoryColors')).toBeTruthy();
```
This way we're making sure the attribute (categoryColors) of the model that triggered the event is not being set to `false`.